magpie-setup-zeppelin

```diff
@@ -0,0 +1,252 @@
+
+if [ "${HBASE_SETUP}" == "yes" ]
+then
+    hbase_jars="${HBASE_HOME}/lib/hbase-annotations-${HBASE_VERSION}.jar"
+    hbase_jars="${HBASE_HOME}/lib/hbase-client-${HBASE_VERSION}.jar,${hbase_jars}"
+    hbase_jars="${HBASE_HOME}/lib/hbase-common-${HBASE_VERSION}.jar,${hbase_jars}"
+    hbase_jars="${HBASE_HOME}/lib/hbase-hadoop2-compat-${HBASE_VERSION}.jar,${hbase_jars}"
+    hbase_jars="${HBASE_HOME}/lib/hbase-it-${HBASE_VERSION}.jar,${hbase_jars}"
+    hbase_jars="${HBASE_HOME}/lib/hbase-prefix-tree-${HBASE_VERSION}.jar,${hbase_jars}"
+    hbase_jars="${HBASE_HOME}/lib/hbase-procedure-${HBASE_VERSION}.jar,${hbase_jars}"
+    hbase_jars="${HBASE_HOME}/lib/hbase-protocol-${HBASE_VERSION}.jar,${hbase_jars}"
+    hbase_jars="${HBASE_HOME}/lib/hbase-resource-bundle-${HBASE_VERSION}.jar,${hbase_jars}"
+    hbase_jars="${HBASE_HOME}/lib/hbase-rest-${HBASE_VERSION}.jar,${hbase_jars}"
+    hbase_jars="${HBASE_HOME}/lib/hbase-server-${HBASE_VERSION}.jar,${hbase_jars}"
+    hbase_jars="${HBASE_HOME}/lib/hbase-shell-${HBASE_VERSION}.jar,${hbase_jars}"
+    hbase_jars="${HBASE_HOME}/lib/hbase-thrift-${HBASE_VERSION}.jar,${hbase_jars}"
+    zeppelin_jars="${hbase_jars},${zeppelin_jars}"
+fi
+
+if [ "${KAFKA_SETUP}" == "yes" ]
+then
+    kafka_jar="${KAFKA_HOME}/libs/kafka_${KAFKA_VERSION}.jar"
+    zeppelin_jars="${zeppelin_jars},${kafka_jar}"
+fi
+
+if [ "${PHOENIX_SETUP}" == "yes" ]
+then
+    phoenix_jars="${PHOENIX_HOME}/phoenix-${PHOENIX_VERSION}-client.jar"
+    phoenix_jars="${PHOENIX_HOME}/phoenix-${PHOENIX_VERSION}-server.jar,$phoenix_jars"
+    phoenix_jars="${PHOENIX_HOME}/phoenix-spark-${PHOENIX_VERSION}.jar,$phoenix_jars"
+    phoenix_jars="${PHOENIX_HOME}/phoenix-core-${PHOENIX_VERSION}.jar,$phoenix_jars"
+    phoenix_jars="${PHOENIX_HOME}/phoenix-server-${PHOENIX_VERSION}.jar,$phoenix_jars"
+    phoenix_jars="${PHOENIX_HOME}/phoenix-server-client-${PHOENIX_VERSION}.jar,$phoenix_jars"
+    zeppelin_jars="${zeppelin_jars},${phoenix_jars}"
+fi
+ZEPPELIN_SPARK_SUBMIT_OPTS="${ZEPPELIN_SPARK_SUBMIT_OPTIONS} --jars $zeppelin_jars"
+
+zeppelinclasspath=${SPARK_HOME}/lib/*:${CLASSPATH}
+
+if [ "${ZEPPELIN_CONF_FILES}X" == "X" ]
+then
+    zeppelinconffiledir=${MAGPIE_SCRIPTS_HOME}/conf
+else
+    zeppelinconffiledir=${ZEPPELIN_CONF_FILES}
+fi
+
+pre_zeppelinenvsh=${zeppelinconffiledir}/zeppelin-env.sh
+pre_zeppelinsitexml=${zeppelinconffiledir}/zeppelin-site.xml
+pre_log4jproperties=${zeppelinconffiledir}/zeppelin.log4j.properties
+
+if [ ! -f "${pre_zeppelinenvsh}" ] \
+    || [ ! -f "${pre_zeppelinsitexml}" ] \
+    || [ ! -f "${pre_log4jproperties}" ]
+then
+    echo "Missing Zeppelin configuration files in ${zeppelinconffiledir}"
+    exit 1
+fi
+
+post_zeppelinenvsh=${ZEPPELIN_CONF_DIR}/zeppelin-env.sh
+post_zeppelinsitexml=${ZEPPELIN_CONF_DIR}/zeppelin-site.xml
+post_log4jproperties=${ZEPPELIN_CONF_DIR}/log4j.properties
+
+SPARK_MASTER="spark://${SPARK_MASTER_NODE}:${SPARK_MASTER_PORT}"
+
```

Comment:
Could we rename this, as SPARK_MASTER is a common environment variable.  I got confused for the need for this, then realized you didn't export it.